entrypoint.sh

```
@@ -0,0 +1,57 @@
+fi
+
+if [ -n "$SSL_CA_PATH" ]; then
+    export CONNECTION_STRING="mysql+mysqlconnector://$DB_USER:$DB_PASS@$DB_HOST:3306/$DB_NAME?ssl_ca=$SSL_CA_PATH"
+else
+    export CONNECTION_STRING="mysql+mysqlconnector://$DB_USER:$DB_PASS@$DB_HOST:3306/$DB_NAME"
+fi
+
+# Check for sentry
+if [ ! -n "$SENTRY_KEY" ]; then
+	echo "Using developer's Sentry DSN for logging, PII will NOT be shared"
+	SENTRY_DSN=https://c8573b9f8ab4d45024aae9909d4353c3@o234159.ingest.us.sentry.io/4508790454681600
```

Shellcheck (0.10.0)

[warning] SENTRY_DSN appears unused. Verify use (or export if used externally).

(SC2034)